data/docs-nav.ts

```diff
@@ -119,6 +119,11 @@ const DocsNav: NavItem[] = [
         linkText: "Settings",
         path: "/docs/running/settings",
       },
+      {
+        title: "Running Grouparoo: Orchestration",
+        linkText: "Orchestration",
```

Comment:
Do our visitors know what orchestration means? I was thinking is this could be more call-to-action driven such as for example "Orchestrate with ETL" (action on subject)

Reply:
I think data engineers (our intended audience) know what the term means, so I think it's a good title.... but I'll add more of an intro on the page